robottelo/config/base.py

```diff
@@ -879,7 +879,7 @@ def _validate_robottelo_settings(self):
         """Validate Robottelo's general settings."""
         validation_errors = []
         browsers = ('selenium', 'docker', 'saucelabs')
-        webdrivers = ('chrome', 'firefox', 'ie', 'phantomjs', 'remote')
+        webdrivers = ('chrome', 'edge', 'firefox', 'ie', 'phantomjs', 'remote')
```

Comment:
And yeah, it's not the same platform and can exist on the same machine simultaneously. Microsoft is positioning Edge as a totally different browser not based on IE (but we know 😄 )